dart/lib/src/scope.dart

```diff
@@ -13,6 +14,14 @@ class Scope {
   /// for example, the route name: `"/users/<username>/"`.
   String? transaction;
 
+  /// Returns active transaction or null if there is no active transaction.
+  ///
+  // I have to set the transaction String from the transaction.name
+  // but the ISentrySpan does not have a name
+  // Java/.NET do not use ISentrySpan but ITransaction
+  // iOS does nit have the transaction String feature
```

Comment:
to discuss

Reply:
discuss with @rhcarvalho , maybe casting it to SentryTracer and accessing the name via setters

Reply:
behavior is correct